src/main/java/stsjorbsmod/powers/BurningPower.java

```diff
@@ -60,12 +60,20 @@ public void playApplyPowerSfx() {
 
     @Override
     public void updateDescription() {
-        if (amount <= 0) {
+        if (this.amount <= 0) {
             this.description = DESCRIPTIONS[4];
-        } else if (this.owner != null && !this.owner.isPlayer) {
-            this.description = DESCRIPTIONS[3] + this.amount + DESCRIPTIONS[1] + (this.amount - this.amount / 2) + DESCRIPTIONS[2] + DESCRIPTIONS[4];
         } else {
-            this.description = DESCRIPTIONS[0] + this.amount + DESCRIPTIONS[1] + (this.amount - this.amount / 2) + DESCRIPTIONS[2] + DESCRIPTIONS[4];
+            int amountToReduceBy;
```

Comment:
Maybe extract this amountToReduceBy calculation to a `calculateNextFalloffAmount` or something so it can be reused in BurningLoseHpAction and keep the relic special case to one spot?

Reply:
Hmm, BurningPower was displaying the number rounded up, and BurningLoseHpAction was using a rounded down value.
Some rewording would allow for some reusable code.

Reply:
I think you could still do it with the current wording if you used the logic here in BurningPower for the calculation and had `BurningLoseHpAction` do `p.amount -= p.calculateNextFalloffAmount()`

Reply:
I'm going to go with `calculateNextBurningAmount()` so I don't have to do endless subtractions.